contrib/packs/actions/expand_repo_name.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import requests
+import json
+import getopt
+import argparse
+import os
+import yaml
+
+from getpass import getpass
+from st2actions.runners.pythonrunner import Action
+
+class expand_repo_name(Action):
+    def __init__(self, config=None):
+        super(expand_repo_name, self).__init__(config=config)
+
+    def run(self, repo_name):
+
+        # Set up the results object
+        results = {}
+
+        try:
+            results['repo_url'] = self.config["repositories"][repo_name]["repo"]
+            results['subtree'] = self.config["repositories"][repo_name]["subtree"]
+        except KeyError:
+            raise ValueError("Missing repositories config for '%s'" % repo_name)
+        else:
+            return results
```

Comment:
Is going 'return results' the best thing to do here?

This ends up with you needing to use '...results.results.repo_url' when you access the results in mistral or the ChatOPS alias. It would be cleaner if it was just '....results.repo'. Am I missing a trick here? I'm not sure where I got that example from, but the duplication of *results* is really jarring.